src/cmd/cmd_query.go

```diff
@@ -231,9 +235,13 @@ func runQueryCmdLine(flags *sybil.FlagDefs) error {
 	for _, v := range allGroups {
 		switch t.GetColumnType(v) {
 		case sybil.STR_VAL:
-			loadSpec.Str(v)
+			if err := loadSpec.Str(v); err != nil {
```

Comment:
i don't get the two statements on one line in an if

Reply:
it's a way to shorten LOC slightly but comes at a slight readability cost.. I can declare the err outside of the switch and deal with it afterwords.